Makefile

```diff
@@ -3,29 +3,29 @@ TRAVIS_BUILD_NUMBER?=debug0
 
 .PHONY: coverage get test clean
 
-all: get coverage AWSnycast
+all: _vendor coverage AWSnycast
 
-AWSnycast: *.go */*.go
-	go build -a -tags netgo -ldflags '-w' .
-	strip AWSnycast
+_vendor: Gomfile
```

Comment:
This isn't so easy, as there's nothing to depend on with my GOPATH (as gom is in ../../../../bin/gom)